lib/test.js

```diff
@@ -21,7 +21,7 @@ var Test = function(options) {
   }
   
   this.web3 = new Web3();
-  this.web3.setProvider(this.sim.provider());
+  this.web3.setProvider(this.sim.provider(options));
```

Comment:
I was actually using whatever is the current version on npm, or was at the time I installed it (see: [DeviateFish/embark-container](https://github.com/DeviateFish/embark-container)), which I believe is v2.2.4.